clang-tools-extra/lsif-clang/package/copy_dependencies.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
+
+set -euo pipefail
+
+if [ "$#" -ne 2 ]; then
+    echo "usage: <binary> <output>"
+    exit 1
+fi
+
+binary="$1"
+output="$2"
+
+for lib in $(ldd "$binary" | cut -d ">" -f 2 | awk '{print $1}' | grep -vE "(linux-vdso\.|ld-linux-x86-64\.|libc\.)"); do
+    cp --verbose "$lib" "$output/"
+done
```

Comment:
We should double-check if the new `libdwarf-dev` and `libelf-dev` dependencies require updating this.

Reply:
Looks like this needed an update for `librt` and `libpthread`; copying them across OSes lead to errors, but removing them fixes the issue. See the commit message of be6a3a11aa2636073b6676b845d0d917afa4d32e for details.

But it works fine otherwise.